src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.Compilation.targets

```diff
@@ -0,0 +1,163 @@
+<Project>
```

Comment:
What's the purpose of splitting this into a new file?

Reply:
That's fine with me. Just FYI the xamlprecompile target is not our ultimate source of truth, we will update this too.